README.md

```diff
@@ -33,11 +33,11 @@ Everything here is written in [Jupyter notebooks](https://jupyter.org/), which m
 If you get stuck with a python question, be like the pros and try www.google.com!  :thought_balloon: If you have a science question,
 leave us a [message](https://github.com/michaelJwilson/DESI-HighSchool/issues/new/) and we'll be happy to help answer them.  But remember, all staff and students at DESI High abide by the [Code of Conduct](https://github.com/michaelJwilson/DESI-HighSchool/blob/master/CODE_OF_CONDUCT.md) :scroll:.  
 
-When you're done, don't forget to get some extra credit by leaving us with what you [liked](https://forms.gle/LGKMVamrtS5StSv56) :star:.  You can show even more support by [donating](berkeleylabfoundation.org/support) a few :dollar: to our support programs for the underprivileged, but make sure to convince your parents first!
+When you're done, don't forget to get some extra credit by leaving us with what you [liked](https://forms.gle/LGKMVamrtS5StSv56) :star:.  You can show even more support by [donating](https://www.berkeleylabfoundation.org/support) a few :dollar: to our support programs for the underprivileged, but make sure to convince your parents first!
```

Comment:
@michaelJwilson I'm not sure if the links are broken on purpose? or if before they were working well as they were... Do you agree with this being fixed?

Reply:
yeh, this is fine.  Not sure I'll wild about pointing there, but it shouldn't be an issue.